living_documentation_generator/generator.py

```diff
@@ -341,32 +342,34 @@ def _generate_issue_summary_table(consolidated_issue: ConsolidatedIssue) -> str:
             consolidated_issue.number,
             consolidated_issue.state.lower(),
             issue_url,
-            # consolidated_issue.created_at,
-            # consolidated_issue.updated_at,
-            # consolidated_issue.closed_at,
+            consolidated_issue.created_at,
+            consolidated_issue.updated_at,
+            consolidated_issue.closed_at,
             labels
         ]
 
-        # Update the summary table, if issue is linked to the project
-        if consolidated_issue.linked_to_project:
-            headers.extend([
-                "Project title",
-                "Status",
-                "Priority",
-                "Size",
-                "MoSCoW"
-            ])
-
-            values.extend([
-                consolidated_issue.project_name,
-                consolidated_issue.status,
-                consolidated_issue.priority,
-                consolidated_issue.size,
-                consolidated_issue.moscow
-            ])
-        else:
-            headers.append("Linked to project")
-            values.append(consolidated_issue.linked_to_project)
+        # Update the summary table, based on the project data mining situation
+        if self.__project_state_mining_enabled:
+            if consolidated_issue.linked_to_project:
+                headers.extend([
+                    "Project title",
+                    "Status",
+                    "Priority",
+                    "Size",
+                    "MoSCoW"
+                ])
+
+                values.extend([
+                    consolidated_issue.project_name,
+                    consolidated_issue.status,
+                    consolidated_issue.priority,
+                    consolidated_issue.size,
+                    consolidated_issue.moscow
+                ])
+            else:
```

Comment:
Is else really needed? I read it several times, and it should be done without dependency on if branch.

Reply:
We have two options that can occur:
1. we have issue extra project data and we show them in the issue page (+ we can figure it out that issue has a project attached)
2. We don't have extra project data + we loose info, if issue has project attached or not. That's why I put else in there and I show extra `Linked to project` in that situation. It is needed in my opinion.

Reply:
Logic simplified in: [458c10b](https://github.com/AbsaOSS/living-doc-generator/pull/27/commits/458c10bb029f1a04947b849a0b7c361827159b0a) still keeping the `else`.